notebooks/book2/03/linreg_post_pred_plot.ipynb

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    MakePlot(SamplePreds, 'linregPostPredSamples', 'functions sampled from posterior')
</pre><p>change <code>figname</code> to <code>figname_latexified</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>